src/MultiBar.js

```diff
@@ -1,21 +1,28 @@
 import React, { Component } from 'react';
 import './styles/MultiBar.css';
 
-export default class SearchBar extends Component {
-  constructor() {
-    super()
-    this.state = {}
+export default class MultiBar extends Component {
+  constructor(props) {
+    super(props)
+    this.state = { multiplayerStatus: null }
   }
-  handleToggle = event => {
-    event.preventDefault();
+  handleToggle = () => {
```

Comment:
This method as well could use another look in the future but great job tackling the functionality.